### PR TITLE
Show error messages

### PR DESCRIPTION
**Pull request by qetiavanashvili**
1.  I have added new properties in viewmodel: isLoading & hasError.
2.  I have written if statement for receiving response (receiveCompletion or receiveValue).

https://github.com/ashalva/ios.memenator/issues/2

### CODE REVIEW

**Review by ashalva — CHANGES_REQUESTED** [review]
You have updated the logic only in vm. What about the view ? Based on the properties you have added you need to show specific subviews in your `DogBreedsListView`. Check how `RedditView` uses these properties and try to replicate.

Your goal is to see:
1. loading when request is being sent
2. error when request fails

**Comment by qetiavanashvili** [comment]
I thought it just needed in vm. I'll see and try to fix it...

File: GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift [hunk 1]
```diff
@@ -13,10 +13,21 @@ struct DogBreedsListView: View {
...
-                Text(dogBreed.name)
-                Text("min life: \(dogBreed.minLife), max life: \(dogBreed.maxLife)")
+        VStack {
+            if viewModel.hasError {
+                Text("Error")
+                    .font(.title)
+                    .padding(.top, 32)
+            } else if viewModel.isLoading {
+                ActivityIndicator(isAnimating: true)
+                Text("Is loading")
```
Comment (ashalva) [inline GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift]: No need to show text here. Just `ActivityIndicator` is enough

Comment (qetiavanashvili) [reply GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift]: I just liked the text and that's why I did it...

Comment (qetiavanashvili) [reply GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift]: I pushed the latest version, if everything is correct, I'll merge

**Review by ashalva — APPROVED** [review]
Looking good!

**Pull request merged**